src/constants/routes.ts

```diff
@@ -0,0 +1,9 @@
+export enum ROUTES {
+	// ----- Market ----- //
+	MARKET = '/market',
+
+	// ----- STAKING ----- //
+	STAKING = '/staking',
+	STAKING_POOLS = '/pools',
+	STAKING_DEPOSITS = '/deposits',
+}
```

Comment:
@colbr @padala @domw30 @akshatmittal We will use global constants for routes instead of hard coding in each component